taskcluster/mozillavpn_taskgraph/transforms/signing.py

```diff
@@ -20,13 +20,14 @@
     "linux/opt",
     "macos/opt",
     "windows/opt",
+    "addons/opt",
 ]
 
 SIGNING_BUILD_TYPES = PRODUCTION_SIGNING_BUILD_TYPES + [
     # Note: it appears we don't have infra for debug signing
     # android builds. Contact releng if you need it :)
     # "android-debug",
-    "addons/opt",
+    # "addons/opt",  # TODO: Add addons debug builds? We have the infra to debug sign them.
```

Comment:
just prod for now. We don't have "debug" builds and for staging we are currently using github.io. We can change this in the future.